faq.html

```diff
@@ -1 +1,153 @@
+                    <div class="btn-container">
+                        
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-4 image-wrapper">
+                <img src="assets/images/mbr-600x600.png" alt="CSAF FAQ">
+            </div>
+        </div>
+    </div>
+</section>
+
+<section data-bs-version="5.1" class="footer2 cid-sRAtK8mSnJ" once="footers" id="footer2-h">
+
+    
+
+    
+
+    <div class="container">
+        <div class="media-container-row align-center mbr-white">
+            <div class="col-12">
+                <p class="mbr-text mb-0 mbr-fonts-style display-7">
+                    © Copyright 2025 OASIS CSAF TC - All Rights Reserved
```

Comment:
Looks like we are a little bit ahead of ourselves. According to my calendar, it is still 2024. :stuck_out_tongue_winking_eye: